app/views/admin/products_v3/_variant_row.html.haml

```diff
@@ -9,13 +9,16 @@
   = error_message_on variant, :sku
 %td
   -# empty
-%td.field
-  = f.text_field :unit_value,
-      value: number_with_precision(variant.unit_value, precision: nil,strip_insignificant_zeros: true),
-      'aria-label': t('admin.products_page.columns.unit_value')
-  = error_message_on variant, :unit_value
-  = f.text_field :display_as, 'aria-label': t('admin.products_page.columns.display_as')
-  = f.button variant.unit_to_display, 'aria-label': t('admin.products_page.columns.unit')
+%td.field.field.on-hand__wrapper{'data-controller': "popout"}
+  = f.button variant.unit_to_display, class: "on-hand__button", 'aria-label': t('admin.products_page.columns.unit'), 'data-popout-target': "button"
+  %div.on-hand__popout{ style: 'display: none;', 'data-controller': 'toggle-control', 'data-popout-target': "dialog" }
```

Comment:
`on_hand` is confusing here. Does it need renaming?

Reply:
For sure.